README.md

```diff
@@ -23,3 +23,11 @@ To make testing close to a real-world scenario, an additional VM reboot step in
 
 > Please don't forget to include respective tests for every new critical feature of the system!<br>
 > See https://www.inspec.io/docs/reference/dsl_inspec/ and existing `/tests` examples which makes easy to add more tests.
+
+### `st2-integration-tests`
+From a user's standpoint, for easier StackStorm troubleshooting there is an `st2-integration-tests` executable shipped in sbin/PATH.<br>
+Sometimes StackStorm is not running properly by some reason.
+Finding an answer is a responsibility of `st2-integration-tests` which will run StackStorm InSpec Tests and report back with more detailed info.
```

Comment:
Discovering why is the responsibility of `st2-integration-tests`...

Reply:
@warrenvw Thanks! fixed